### PR TITLE
docs(about us): update contributors list and numbers

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Bump number of contributors to `830`
- Highlight contributions of more users
- Move myself back to active contributor

## Context

I think we can highlight more people. 🥳

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/about-us.md [hunk 1]
```diff
@@ -26,18 +26,22 @@ Next up, we have these people who help maintain parts of Renovate:
...
 Some features made a lot of people happy, and efficient!
 
-- [@HonkingGoose](https://github.com/HonkingGoose) worked on the docs and was community manager
 - [@ikesyo](https://github.com/ikesyo) regularly helpful
 - [@astellingwerf](https://github.com/astellingwerf) reviews PRs
 - [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
 - [@shegox](https://github.com/shegox) worked on the Go manager, and improved our docs
 - [@setchy](https://github.com/setchy) focused on Bitbucket Cloud and replacement features
+- [@cgrindel](https://github.com/cgrindel) created the `bazel-module` manager
+- [@RahulGautamSingh](https://github.com/RahulGautamSingh) refactored our code and ...
```
Comment (HonkingGoose) [inline docs/usage/about-us.md:42]: @RahulGautamSingh I see you refactored a lot of our code, but don't know what to highlight extra (special feature/good bugfix).

File: docs/usage/about-us.md [hunk 2]
```diff
@@ -26,18 +26,22 @@ Next up, we have these people who help maintain parts of Renovate:
...
+- [@Gabriel-Ladzaretti](https://github.com/Gabriel-Ladzaretti) ...
```
Comment (HonkingGoose) [inline docs/usage/about-us.md:43]: @Gabriel-Ladzaretti Let me know what you'd like to be credited for here. I see you landed enough commits to be included in this list (if you want). 😉

File: docs/usage/about-us.md [hunk 3]
```diff
@@ -26,18 +26,22 @@ Next up, we have these people who help maintain parts of Renovate:
...
+- [@Churro](https://github.com/Churro) worked on the `gradle` manager
```
Comment (HonkingGoose) [inline docs/usage/about-us.md:44]: @Churro please check if you're happy with this credit on our About Us page. 😉

**Review by HonkingGoose — COMMENTED** [review]
Pinging the mentioned users for their review.

File: docs/usage/about-us.md (hunk 1 above)
Comment (RahulGautamSingh) [reply docs/usage/about-us.md:42]: I think the highlight of my commits will be perf improvements after reduced cloning during update and onboarding(ongoing).

File: docs/usage/about-us.md (hunk 3 above)
Comment (Churro) [inline docs/usage/about-us.md:44]: Thanks for the credits! 

Looking at my PRs, I'd suggest the following addition: 
```suggestion
- [@Churro](https://github.com/Churro) worked on the `gradle` manager and added support for OSV vulnerabilities
```

Comment (Churro) [reply docs/usage/about-us.md:44]: Thanks for the credits!

Looking at my PRs, I'd suggest the following addition:

```suggestion
- [@Churro](https://github.com/Churro) worked on the `gradle` manager and added support for OSV vulnerabilities
```

**Review by cgrindel — DISMISSED** [review]

**Review by Churro — DISMISSED** [review]

File: docs/usage/about-us.md (hunk 1 above)
Comment (HonkingGoose) [reply docs/usage/about-us.md:42]:
```suggestion
- [@RahulGautamSingh](https://github.com/RahulGautamSingh) refactored a lot of code and worked on performance improvements like reduced cloning during updates and onboarding
```

How about this?

**Review by RahulGautamSingh — DISMISSED** [review]

File: docs/usage/about-us.md [hunk 4]
```diff
@@ -26,18 +26,22 @@ Next up, we have these people who help maintain parts of Renovate:
...
-- [@HonkingGoose](https://github.com/HonkingGoose) worked on the docs and was community manager
 - [@ikesyo](https://github.com/ikesyo) regularly helpful
 - [@astellingwerf](https://github.com/astellingwerf) reviews PRs
 - [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
 - [@shegox](https://github.com/shegox) worked on the Go manager, and improved our docs
 - [@setchy](https://github.com/setchy) focused on Bitbucket Cloud and replacement features
+- [@cgrindel](https://github.com/cgrindel) created the `bazel-module` manager
+- [@RahulGautamSingh](https://github.com/RahulGautamSingh) refactored a lot of code and worked on performance improvements like reduced cloning during updates and onboarding
+- [@Gabriel-Ladzaretti](https://github.com/Gabriel-Ladzaretti) ...
+- [@Churro](https://github.com/Churro) worked on the `gradle` manager and added support for OSV vulnerabilities
```
Comment (rarkins) [inline docs/usage/about-us.md:44]: I think @Churro should be added to the regular section

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/usage/about-us.md (hunk 2 above)
Comment (Gabriel-Ladzaretti) [reply docs/usage/about-us.md:44]: Thanks a lot for this :), 
I really liked working on some features such as s3 backed repo cache, implementing our own process spawn and the config migration PR but there is no single focus point for me. so i guess something like "regularly helpful" might fit ? ¯\\_(ツ)_/¯

File: docs/usage/about-us.md [hunk 5]
```diff
@@ -26,18 +26,22 @@ Next up, we have these people who help maintain parts of Renovate:
...
 
-- [@HonkingGoose](https://github.com/HonkingGoose) worked on the docs and was community manager
 - [@ikesyo](https://github.com/ikesyo) regularly helpful
 - [@astellingwerf](https://github.com/astellingwerf) reviews PRs
 - [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
 - [@shegox](https://github.com/shegox) worked on the Go manager, and improved our docs
 - [@setchy](https://github.com/setchy) focused on Bitbucket Cloud and replacement features
+- [@cgrindel](https://github.com/cgrindel) created the `bazel-module` manager
+- [@RahulGautamSingh](https://github.com/RahulGautamSingh) refactored a lot of code and worked on performance improvements like reduced cloning during updates and onboarding
+- [@Gabriel-Ladzaretti](https://github.com/Gabriel-Ladzaretti) ...
```
Comment (rarkins) [inline docs/usage/about-us.md:44]:
```suggestion
- [@Gabriel-Ladzaretti](https://github.com/Gabriel-Ladzaretti) S3 repo cache, child process refactoring, others
```

File: docs/usage/about-us.md (hunk 2 above)
Comment (HonkingGoose) [reply docs/usage/about-us.md]: I see the maintainer pushed a commit that describes your work. Let me know if you like it. 😉

**Review by Gabriel-Ladzaretti — APPROVED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**